src/components/cc-pricing-product/cc-pricing-product.js

```diff
@@ -67,163 +69,455 @@ export class CcPricingProduct extends LitElement {
     /** @type {Currency} Sets the currency used to display the prices (defaults to euros). */
     this.currency = CURRENCY_EUR;
 
-    /** @type {string|null} Sets the description of the product (can be overriden with the default slot). */
-    this.description = null;
+    /** @type {PricingProductState} Sets the state of the pricing product component. */
+    this.product = { state: 'loading' };
 
-    /** @type {boolean} Displays an error message. */
-    this.error = false;
+    /**
+    * @type {Temporality[]} Sets the time window(s) you want to display the prices in (defaults to 30 days with 2 fraction digits).
+    * You may display one or several time windows at once.
+    */
+    this.temporalityList = DEFAULT_TEMPORALITY_LIST;
```

Comment:
question: `temporalityList` is documented as "time window(s)", should we rename `temporalityList` to `timeWindowList`?